metron-interface/pom.xml

```diff
@@ -25,6 +25,9 @@
     </parent>
     <description>Interfaces for Metron</description>
     <url>https://metron.apache.org/</url>
+    <properties>
+        <jackson.version>2.9.5</jackson.version>
```

Comment:
What's this used for?

Reply:
It's used in both metron-rest and metron-rest-client.  We are using several different jackson libraries.

Reply:
Ok, do me a favor, when you test this out in full-dev, make sure that you can continue to use the REST endponts backing the management UI.  We often have issues with classpath issues around newer versions of Jackson.  Should we exclude the other jackson dependencies coming in transitively in metron-rest and metron-rest-client?

Reply:
I tested the other endpoints and everything seems to still be functioning.  I also inspected the mvn dependency tree and it looks like the jackson version is consistent across dependencies.